roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,145 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DockerImageAvailability(OpenShiftCheck):
+    """Check that required Docker images are available.
+
+    This check attempts to pull required docker images and fail if
+    any image is missing or an error occurs during pull.
+    """
+
+    name = "docker_image_availability"
+    tags = ["preflight"]
+
+    skopeo_image = "juanvallejo/skopeo:v3.4"
```

Comment:
I am not sure when we will introduce the skopeo binary to our `openshift/openshift/ansible`, but assuming that we can count on Skopeo being in that image in the near-future, I can replace `juanvallejo/skopeo:v3.4` with our ansible image. The trick would be the image version. I only see we have (at least in docker hub) the `latest` tag available: https://hub.docker.com/r/openshift/openshift-ansible/tags/

cc @brenton